phastSim/phastSim.py

```diff
@@ -607,14 +693,96 @@ def normalize(node, norm):
                 node.rate /= norm
                 if node.isTerminal:
                     node.rates /= norm
+
                 else:
                     normalize(node.belowNodes[0], norm)
                     normalize(node.belowNodes[1], norm)
 
         # normalize the rates
         normalize(node=self.genomeRoot, norm=norm)
 
+        if self.indels:
+            # if there are indels then some further calculation is require to convert the rates from 
+            # substitution rates to overall rates.
+            
+
+            def include_indels(node, scale):
+                
+                if node.isTerminal:
+                    node.insertionRate * scale
```

Comment:
Shouldn't this be "*=" instead of just "*" ?

Reply:
sorry, the asterix disappeared, I mean *= instead of =

Reply:
Same in the line below

Reply:
Oops - Yes! Thanks